### PR TITLE
Long post feature

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Handle the case when the post height is above 1000px and treats it as a long post (the measurement does not take into account link previews, images or reactions.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-9638

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Has UI changes
- [x] Includes text changes and localization file updates

#### Device Information
This PR was tested on:
* Android 7.1.1 Nexus 5 (Emulator)
* iOS 11.2 iPhone 5s and iPhone X (Simulators)
* iOS 11.2.6 iPhone 6

#### Screenshots
Android
screenshot_1521525905

screenshot_1521525923

screenshot_1521525934

iOS
simulator screen shot - iphone x - 2018-03-20 at 08 05 41

simulator screen shot - iphone x - 2018-03-20 at 08 05 54

simulator screen shot - iphone x - 2018-03-20 at 08 06 01

### CODE REVIEW

File: app/components/reactions/reactions.js [hunk 1]
```diff
@@ -69,45 +74,68 @@ export default class Reactions extends PureComponent {
 
         const addMoreReactions = (
             <TouchableOpacity
+                key='addReaction'
                 onPress={this.props.onAddReaction}
                 style={[styles.reaction]}
             >
-                <Text style={styles.more}>{'+'}</Text>
+                <Image
+                    source={addReaction}
```
Comment (saturninoabril) [inline app/components/reactions/reactions.js]: On first read, I thought its an action. Can we change it to `addReactionIcon`?

Comment (enahum) [reply app/components/reactions/reactions.js]: Sure thing, I’ll do it as soon as I can.

**Comment by enahum** [comment]
@saturninoabril done

**Comment by enahum** [comment]
Sure, only in the preview window, right?

**Comment by enahum** [comment]
@asaadmahmood changes made, screenshots attached
screenshot_1521983801

screenshot_1521983808

simulator screen shot - iphone x - 2018-03-25 at 15 34 30

simulator screen shot - iphone x - 2018-03-25 at 15 57 39

**Comment by asaadmahmood** [comment]
@enahum The image icons are not centered vertically. Also the thumbnail (which is where the icon or image is shown) should maintain an aspect ratio of 1:1. So its width should equal its height.
screen shot 2018-03-25 at 6 47 51 pm

**Comment by enahum** [comment]
Hmm yes somehow on the long post is being rendered differently, I’ll take another look

**Comment by enahum** [comment]
@asaadmahmood this should be good now

screenshot_1521986206

screenshot_1521986224

image

simulator screen shot - iphone x - 2018-03-25 at 16 54 39

simulator screen shot - iphone x - 2018-03-25 at 16 54 43

image

**Pull request merged**